doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,17 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-ignore_unexpected_eof>
+
+Some TLS implementations do not send close_notify on shutdown. Before OpenSSL
+3.0, SSL_ERROR_SYSCALL with errno=0 was reported in such cases. This
+behaviour contradicts the B<SSL_ERROR_SYSCALL> documentation. Since OpenSSL 3.0
+the returned error is B<SSL_ERROR_SSL> with a meaningful error on the error
+stack. This option gives the B<SSL_ERROR_ZERO_RETURN> error in such cases.
```

Comment:
I suggest you make that something like: When this option is enabled an B<SSL_ERROR_ZERO_RETURN> error is returned in case an EOF is detected and the close_notify alert was not received.

Reply:
Done. Thanks!